Yafc.Model/Blueprints/Blueprint.cs

```diff
@@ -98,17 +98,44 @@ public void Set(IObjectWithQuality<Goods> goods) {
 
 [Serializable]
 public class BlueprintEntity {
-    [JsonPropertyName("entity_number")] public int index { get; set; }
+    [JsonPropertyName("entity_number")]
+    public int index { get; set; }
+
     public string? name { get; set; }
     public string? quality { get; set; }
     public BlueprintPosition position { get; set; } = new BlueprintPosition();
     public int direction { get; set; }
+
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? recipe { get; set; }
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? recipe_quality { get; set; }
-    [JsonPropertyName("control_behavior")] public BlueprintControlBehavior? controlBehavior { get; set; }
+
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    [JsonPropertyName("control_behavior")]
+    public BlueprintControlBehavior? controlBehavior { get; set; }
     public BlueprintConnection? connections { get; set; }
-    [JsonPropertyName("request_filters")] public List<BlueprintRequestFilter> requestFilters { get; } = [];
+
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    [JsonPropertyName("request_filters")]
+    public BlueprintRequestFilterSections? requestFilters { get; set; }
     public List<BlueprintItem> items { get; } = [];
+    [JsonPropertyName("burner_fuel_inventory")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
```suggestion
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    [JsonPropertyName("burner_fuel_inventory")]
```

Same order as the others (I know it is a nitpick...)